Makefile

```diff
@@ -98,13 +98,11 @@ clean-launch:
 integration-tests:
 	go test ./src/server -timeout 120s
 
+protofix:
+	docker build -t pachyderm/protofix -f Dockerfile.protofix .
+
```

Comment:
Can this be `docker-build-proto` to match our other targets that build docker images